### DISCUSSION TITLE
https://modelviewer.dev/docs/ - Page not Loading

### DISCUSSION THREAD

**Discussion by jayasurya00098** [google/model-viewer#3698] [Q&A] [answered]
I've been trying to access the documentation of Model Viewer but for some reason, it is simply blank.
![1](https://user-images.githubusercontent.com/17214071/183762971-ccba81c7-c586-4504-a74e-5ed04889ae89.png)

**Comment by elalish** [comment, marked as answer]
Yeah, sorry, this was just fixed in #3696. Should be deployed shortly.

**Reply by jayasurya00098** [reply]
Thanks, @elalish
